xbmc/cores/VideoPlayer/VideoRenderers/BaseRenderer.h

```diff
@@ -92,6 +92,13 @@ class CBaseRenderer
 
   void SetVideoSettings(const CVideoSettings &settings);
 
+  // Get debug info from render buffer
+  virtual void GetDebugInfo(int idx,
```

Comment:
I think a returning defined struct of `std::string` would be better suited here and more future proof.